### ISSUE TITLE
Proposal: Introduce Election as a sub-type of Event

### ISSUE THREAD

**Issue by mrwilson** [schemaorg/schemaorg#2391]
Hi,

I'm a contributor to [Democracy Club](https://democracyclub.org.uk) - we're a group of volunteers that crowd-source data for elections that are run in the UK. We run [WhereDoIVote](https://wheredoivote.co.uk) and [WhoCanIVoteFor](https://whocanivotefor.co.uk) to expose this data openly.

We've recently started marking up our elections as [`Event`](https://schema.org/Event) but a lot of the crucial information about the election is lost.

For example, we would like:
* To attach `candidates` as a list of `Person`/`Organization` (depending on the method of voting)
* To attach `results` after the fact. There's already something similar in [`VoteAction`](https://schema.org/VoteAction) but this doesn't appear to have capacity for the total vote count.

I'm imagining something like below, modified from the live instance of WhoCanIVoteFor:

```json
{
    "@context": "http://schema.org/",
    "@type": "Election",
    "startDate": "2019-12-12 07:00",
    "endDate": "2019-12-12 22:00",
    "name": "UK Parliament Election - Example",
    "description": "Election of the Member of Parliament for Example",
    "location" : {
        "@type" : "Place",
        "address": "Example, UK"
    },
    "candidates": [{
        "@context": "http://schema.org/",
        "@type": "Person",
        "name": "Foo Bar",
        "worksFor": { "@type": "Organization", "name": "Foo Bar Party }
     }]
}
```

This will hopefully get closed and onted - if there's no initiatives underway to propose something like this, I'm happy to go through the process to make a formal proposal.

Thanks in advance!

**Comment by DmPo** [comment]
Would also be great to have something like ParallelVoteCount / ExitPoll
/ ElectoralFraud.

Best regards,
Dmytro

**Comment by mrwilson** [comment]
Nice. There might be scope for a broader set of voting/elections/democracy-related "stuff" -  I figured that `Election` was a suitably narrow concept to start off with.

**Comment by DmPo** [comment]
Yes. Also have a look at the ClaimReview used to describe fake news.

In essence, falsified electoral results - when reported - is a type of fake
news, so ideally the two things are compatible.

Best regards,
Dmytro

-- 
Dmytro Potekhin
Founder & CEO
FakesKiller.com
dp@fakeskiller.com

Ср, 13 лист. 2019, 15:28 користувач Alex Wilson <notifications@github.com>
пише:

**Comment by jenifertorro777** [comment]
I'm still confused I need help
